mod `WaitLoad` to `WaitStable` (#101)
go-rod/go-rod.github.io: get-started/README.md

func main() {
    page := rod.New().MustConnect().MustPage("https://www.wikipedia.org/")
    page.MustWaitStable().MustScreenshot("a.png")
}
```

The `rod.New` creates a browser object, the `MustConnect` launches and connects to a browser.
The `MustPage` creates a page object, it's like a page tab in the browser. The `MustWaitStable`

> waits for the page is fully loaded. The `MustScreenshot` takes a screenshot of the page.

Reviewer: ```suggestion
waits until the page rarely change. The `MustScreenshot` takes a screenshot of the page.
```
